main.go

```diff
@@ -67,6 +67,7 @@ func main() {
 	flag.BoolVar(&showHelp, "h", false, "print usage")
 	flag.BoolVar(&showVersion, "version", false, "print version")
 	flag.StringVar(&configFile, "c", "", "config file (i.e. -c vulcan.yaml)")
+	flag.StringVar(&cfg.Conf.Policy, "p", "", "policy to execute")
```

Comment:
Does it makes sense of applying more than 1 policy?

Reply:
It makes sense. As discussed, with the current implementation of policies it should be easy to add this functionality at any moment.